### DISCUSSION TITLE
Explore the compose feature

### DISCUSSION THREAD

**Discussion by danielaki** [acheong08/EdgeGPT#151] [General]
Testing some prompts commands, I was able to make it do things it doesn't do anymore in Bing Chat. And the best part: as I generated new models, it mixed them with previous dialogues. So, is it possible to reverse engineer a chat?

![image](https://user-images.githubusercontent.com/126002925/221200454-72396f2e-f38f-4236-9b65-0acb3adf14cc.png)

**Comment by acheong08** [comment]
Oh this is new. Will reverse when I have the time as a separate module

**Comment by LuckyHead11** [comment]
@danielaki How do you get to that screen? I have the insider Microsoft edge and access to BingGPT.

**Reply by hrizal** [reply]
open edge and click bing icon on the right.

> Microsoft Edge
> Version 111.0.1661.62 (Official build) (64-bit)

**Comment by danielaki** [comment]
@LuckyHead11 browser edge canary... 112.0.1704.0 + works

**Comment by scmanjarrez** [comment]
I read about that some time ago, more info here https://twitter.com/vaibhavk97/status/1624926346076889089

**Comment by acheong08** [comment]
I don't think I'll have the time to implement it. Been a bit too busy
